README.md

```diff
@@ -11,18 +11,21 @@ This tool can be viewed from two perspectives: first from the perspective of a u
 To generate the documentation for almost any ROS 2 package, first build the package locally, then run this command replacing the arguments with appropriate directories:
 
 ```
-$ rosdoc2 build \
+rosdoc2 build \
   --package-path ./src/path/to/my_package_name
 ```
 
+> [!NOTE]
+> Please make sure that ROS 2 colcon workspace is NOT sourced during build.
```

Comment:
On second thought I'm not sure if we should make this generalization for reasons explained here https://github.com/ros-infrastructure/rosdoc2/issues/66#issuecomment-2001897492.

How about we create a `Known Issues` section at the bottom of the README and link https://github.com/ros-infrastructure/rosdoc2/issues/68?

Reply:
sgtm.